Monika After Story/game/script-affection.rpy

```diff
@@ -1766,10 +1874,6 @@ label monika_affection_nickname:
             "okasa"
         ]
 
-        # unlock this event
-        aff_nickname_ev = mas_getEV("monika_affection_nickname")
```

Comment:
Either re-add this line here or handle the topic rename in another way on the python block with the comment `# change the prompt for this event` below.